pkg/jobs/jobspb/jobs.proto

```diff
@@ -1437,6 +1445,7 @@ message Payload {
     HistoryRetentionDetails history_retention_details = 47;
     LogicalReplicationDetails logical_replication_details = 48;
     UpdateTableMetadataCacheDetails update_table_metadata_cache_details = 49;
+    StandbyReadTSPollerDetails standby_read_ts_poller_details = 50;
```

Comment:
Nit: can we update the Next ID comment at the bottom of the message? Looks like it has fallen behind but we can fix that while we're here.